expr/test.go

```diff
@@ -9,6 +9,26 @@ import (
 
 var results []models.Series
 
+func equalTags(exp, got []models.Series) error {
+	for i, g := range got {
+		o := exp[i]
```

Comment:
exp and got may have a different len?

Reply:
Generally already checked, but I can add the check here.